utils/getOffchainPrice.ts

```diff
@@ -30,6 +30,22 @@ function _getBitstampPriceFromJSON(jsonData: any) {
   return Number(jsonData.last);
 }
 
+// This is needed because our new cors proxy wont forward http or https. We really only use https.
+function stripProtocol(url: string) {
+  return url.replace("https://", "");
+}
+
+// Small helper to wrap a url in cors proxy.
+export function proxyUrl(
+  url: string,
+  // This env can be set with a .env file. It has to be prefixed with NEXT_PUBLIC_ to be available in browser
+  // This has a default set for now until we can get prod env var in.
+  proxy = process.env.NEXT_PUBLIC_CORS_PROXY_URL ||
```

Comment:
i tested this. and it does inject env in if set. left in hardcoded string so it works in production